drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/main/java/org/drools/workbench/services/verifier/plugin/client/builders/ValuesResolver.java

```diff
@@ -130,11 +130,19 @@ private String getType() throws
         }
     }
 
+    private Boolean getBoolean() {
+        if (operatorFromCell.isPresent()) {
+            return tryBoolean(getCellValueFromCellWithOperator());
```

Comment:
See my comment about `tryBoolean`

Reply:
I kept the name tryBoolean since can't think a better name. It can return true, false or null.